src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -1855,555 +1635,36 @@ void GetUserConvectionCoefficients(EnergyPlusData &state)
-
-        i += 2;
-        state.dataConvectionCoefficient->InsideFaceAdaptiveConvectionAlgo.MixedStableCeilingEqNum =
-            SetInsideAdaptiveConvectionAlgo(state,
-                                            HcInt_ConvectionTypesMap,
-                                            ErrorsFound,
-                                            state.dataIPShortCut->cAlphaArgs(i),
-                                            state.dataIPShortCut->cAlphaArgs(i + 1),
-                                            state.dataIPShortCut->cAlphaFieldNames(i),
-                                            state.dataIPShortCut->cAlphaFieldNames(i + 1),
-                                            RoutineName,
-                                            CurrentModuleObject);
-
-        i += 2;
-        state.dataConvectionCoefficient->InsideFaceAdaptiveConvectionAlgo.MixedUnstableCeilingEqNum =
-            SetInsideAdaptiveConvectionAlgo(state,
-                                            HcInt_ConvectionTypesMap,
-                                            ErrorsFound,
-                                            state.dataIPShortCut->cAlphaArgs(i),
-                                            state.dataIPShortCut->cAlphaArgs(i + 1),
-                                            state.dataIPShortCut->cAlphaFieldNames(i),
-                                            state.dataIPShortCut->cAlphaFieldNames(i + 1),
-                                            RoutineName,
-                                            CurrentModuleObject);
-
-        i += 2;
-        state.dataConvectionCoefficient->InsideFaceAdaptiveConvectionAlgo.MixedWindowsEqNum =
-            SetInsideAdaptiveConvectionAlgo(state,
-                                            HcInt_ConvectionTypesMap,
-                                            ErrorsFound,
-                                            state.dataIPShortCut->cAlphaArgs(i),
-                                            state.dataIPShortCut->cAlphaArgs(i + 1),
-                                            state.dataIPShortCut->cAlphaFieldNames(i),
-                                            state.dataIPShortCut->cAlphaFieldNames(i + 1),
-                                            RoutineName,
-                                            CurrentModuleObject);
+                                                                 ipsc->lNumericFieldBlanks,
+                                                                 ipsc->lAlphaFieldBlanks,
+                                                                 ipsc->cAlphaFieldNames,
+                                                                 ipsc->cNumericFieldNames);
+        // state.dataConvectionCoefficient->intAdaptiveConvAlgo.Name = ipsc->cAlphaArgs(1); // not used by E+, unique object
+
+	for (int iInConvClass = 0, i = 2; iInConvClass < static_cast<int>(IntConvClass::Num) && i <= NumAlphas; ++iInConvClass, i+=2) {
```

Comment:
Tada!

Reply:
Why is `int i = 2;` not needed?

Reply:
In a variable declaration, you can declare multiple variables of the same type (here `int`) separated by commas.